petl/util/timing.py

```diff
@@ -35,16 +37,49 @@ def progress(table, batchsize=1000, prefix="", out=sys.stderr):
     return ProgressView(table, batchsize, prefix, out)
 
 
+def log_progress(table, batchsize=1000, prefix="", out=None, level=logging.INFO):
```

Comment:
Suggest to rename 'out' to 'logger' just to help make it clear that a logger is expected here.